charts/stable/metallb/templates/_bgpadvertisement.tpl

```diff
@@ -0,0 +1,41 @@
+{{- define "metallb.bgpadv" -}}
+
+{{- if .Values.metallb.BGPAdvertisements }}
+{{- range .Values.metallb.BGPAdvertisements }}
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: {{ .name }}
```

Comment:
Why?
We always used our own includes for names on all other apps?
Don't blindly start copying things.

I notice this a lot to be honest... Upstream sources are inspiration, not copy-pasta material!

Reply:
the name in this case is a user-specified one.
eg. a user can have multiple bgpadvertisements for local and external
At least this is what I can understand from their docs and examples
https://metallb.universe.tf/configuration/_advanced_bgp_configuration/

Reply:
ohh shot, too early for me sorry... I was reading `.Name` as in: deployed helm name